dashboards/google-cloud-composer/cloud-composer-monitor.json

```diff
@@ -0,0 +1,389 @@
+{
+  "category": "CUSTOM",
+  "dashboardFilters": [
+    {
+      "filterType": "RESOURCE_LABEL",
+      "labelKey": "project_id",
+      "templateVariable": "Project"
+    },
+    {
+      "filterType": "RESOURCE_LABEL",
+      "labelKey": "environment_name",
+      "templateVariable": "Cluster"
+    }
+  ],
+  "displayName": "sb-data-projects-composer-load-test-v1",
+  "etag": "07c4608fd0b2623bfdb5c444238ba2fb",
+  "labels": {},
+  "mosaicLayout": {
+    "columns": 12,
+    "tiles": [
+      {
+        "height": 1,
+        "widget": {
+          "text": {
+            "content": "",
+            "format": "RAW"
+          },
+          "title": "COMPOSER MONITOR"
```

Comment:
Can this be "Cloud Composer Monitor" instead?

Reply:
changed!